samples/ios/Continote/Continote/Note.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Copyright (c) 2017 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import FirebaseDatabase
+
+/**
+ A Note for the user within Continote.
+ The schema of each Note is outlined in sample-database.rules.json within the web sample.
+
+ Each Note is considered immutable on the client-side to simplify syncing with the database for
+ this sample.
+ */
+struct Note {
+
+  // The title of this Note.
+  let title: String
+
+  // The main content of this Note.
+  let content: String
+
+  // The Firebase Realtime Database representation of this Note.
+  // This could be used to add or set a Note value in the database.
+  let asFirebaseData: [String : AnyObject]
```

Comment:
Hmm yeah I think I wrote this before I realized everything was a let constant. I'm fine leaving it as is in this case, since multiple calls to `firebaseData` would be faster. Also, as you currently have it implemented it'd be easy to see when if you add a new parameter. Only thing now is to rename it to `firebaseData` :)